actualbudget/actual
# fix: notes is (nothing) not working

Testing:
1. create the following rule
![image](https://github.com/user-attachments/assets/a96249ec-9a99-423f-a777-274cc1ffd586)

2. create a new transaction, and set the payee
3. the notes should be set.
